internal/testutil/test_hcls.go

```diff
@@ -101,6 +102,8 @@ const DedicatedCloudShouldCreateLag = false
 const DedicatedCloudEncap = "qinq"      // Azure only
 const DedicatedCloudPortCat = "primary" // Azure only
 
+const LinkAggGroupInterval = "fast"
```

Comment:
because this file is becoming use, could you add the following comment before the const?

`// packetfabric_link_aggregation_group`